### PR TITLE
Add Heart-Beat animation

### PR DESCRIPTION
**Pull request by jamesgeorge007**
This is in reference to the issue #829 which deals about adding a Heart-Beat animation.
> Check out the demo [here](https://codepen.io/jamesgeorge/pen/YjyBZw)

- [x] Modified animate-config.json to include the heartBeat animation.
- [x] Add hearBeat.css to the source directory within attention-seekers.
- [x] Update Readme to include heartBeat class within the list of classes provided.

### CODE REVIEW

**Comment by phlegx** [comment]
Nice! Thank you!

**Comment by phlegx** [comment]
Can this PR please be merged in a new release?

**Comment by WarenGonzaga** [comment]
We will review it first :) @eltonmesquita can you please have a look on this?

**Comment by eltonmesquita** [comment]
Hey @daneden . I'd love to hear your opinion about this one ;D

**Comment by WarenGonzaga** [comment]
@eltonmesquita and @daneden actually I am using this one with the latest feature :) I agree to add this one... but I am checking with the code if it will have a conflict or not following the way @daneden write animations.

**Comment by daneden** [comment]
I like it! ❤️

We actually don't want to add the `webkit` rules to the source: we only need the unprefixed CSS.

**Comment by WarenGonzaga** [comment]
@daneden will look in the code and fix it and submit another PR :) need to finish the feature first before that :) thanks for being active today @daneden :)

**Comment by eltonmesquita** [comment]
Cool @daneden ! @jamesgeorge007 , I'll happily merge this when you remove the  `-webkit` parts.

**Comment by jamesgeorge007** [comment]
@eltonmesquita Sure, Thank you.

**Comment by jamesgeorge007** [comment]
@eltonmesquita @daneden  I have removed the ```-webkit``` prefixes from the file within the source directory. 
*Hope this solves the issue with those conflicts that @eltonmesquita mentioned above.*

**Comment by WarenGonzaga** [comment]
@jamesgeorge007 alright well done... @eltonmesquita please merge this one...

File: animate-config.json [hunk 1]
```diff
@@ -9,7 +9,8 @@
     "swing": true,
     "tada": true,
     "wobble": true,
-    "jello": true
+    "jello": true,
+    "hearBeat": true
```
Comment (daneden) [inline animate-config.json]: We actually need to fix this typo first!

**Comment by jamesgeorge007** [comment]
@daneden Typos fixed! :+1:

**Comment by daneden** [comment]
Awesome! I actually just noticed that `animate.min.css` hasn't been updated in this change: can you run `gulp` from the command line to make sure that's also updated?

**Comment by jamesgeorge007** [comment]
@daneden I'm not sure what lead to conflicts. I just installed the dependencies as listed within the package.json file and ran ```npm start``` from the terminal which triggered ```gulp``` adding heartBeat animation to the animate.min.css file. What went wrong ?

**Comment by eltonmesquita** [comment]
@jamesgeorge007 your branch has just become outdated because of some merges that happened before. Just build and commit it and everything will be ok.

**Comment by jamesgeorge007** [comment]
@daneden @eltonmesquita Hope everythings fine now :smiley:

**Comment by daneden** [comment]
This is adding `package-lock.json` and some changes to `package.json` too, but I also ran into those when I recently updated. I'm ok merging since we'll be coming up on a 3.7.0 release soon (probably this weekend!)

**Pull request merged**